.github/actions/orca/orca-python-py37-spark3-action/nightly-test/action.yml

```diff
@@ -17,6 +17,17 @@ runs:
         # install orca
         pip install -i https://pypi.org/simple python/orca/src/dist/bigdl_orca*-py3-none-manylinux1_x86_64.whl
 
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} torch==1.7.1
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} torchvision==0.8.2
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} torchmetrics==0.10.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow==1.15.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-gan==2.0.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-probability==0.7.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-datasets==3.2.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} pyarrow==4.0.1
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} pandas
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} h5py==2.10.0
```

Comment:
need to check which example uses these three packages

Reply:
h5py:
~~~
=========================== short test summary info ============================
FAILED test/bigdl/orca/learn/spark/test_estimator_keras_for_spark.py::TestEstimatorForKeras::test_estimator_keras_save_load - AttributeError: 'str' object has no attribute 'decode'
FAILED test/bigdl/orca/learn/spark/test_estimator_keras_for_spark.py::TestEstimatorForKeras::test_estimator_keras_weights_save_load - AttributeError: 'str' object has no attribute 'decode'
FAILED test/bigdl/orca/learn/spark/test_estimator_keras_for_spark.py::TestEstimatorForKeras::test_submodel_in_keras_squential - AttributeError: 'str' object has no attribute 'decode'
~~~
test link:
https://github.com/intel-analytics/BigDL/actions/runs/3880476698/jobs/6618566774

Reply:
> h5py:
> 
> ```
> =========================== short test summary info ============================
> FAILED test/bigdl/orca/learn/spark/test_estimator_keras_for_spark.py::TestEstimatorForKeras::test_estimator_keras_save_load - AttributeError: 'str' object has no attribute 'decode'
> FAILED test/bigdl/orca/learn/spark/test_estimator_keras_for_spark.py::TestEstimatorForKeras::test_estimator_keras_weights_save_load - AttributeError: 'str' object has no attribute 'decode'
> FAILED test/bigdl/orca/learn/spark/test_estimator_keras_for_spark.py::TestEstimatorForKeras::test_submodel_in_keras_squential - AttributeError: 'str' object has no attribute 'decode'
> ```
> 
> test link: https://github.com/intel-analytics/BigDL/actions/runs/3880476698/jobs/6618566774

pandas:
~~~
python -m pytest -v test/bigdl/orca/learn/spark --ignore=test/bigdl/orca/learn/spark/test_estimator_openvino.py
TestEstimatorForKeras.test_xshards_spark_estimator

E                   py4j.protocol.Py4JJavaError: An error occurred while calling z:org.apache.spark.api.python.PythonRDD.collectAndServe.
E                   : org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 286.0 failed 1 times, most recent failure: Lost task 0.0 in stage 286.0 (TID 240) (localhost executor driver): org.apache.spark.api.python.PythonException: Traceback (most recent call last):
E                     File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/worker.py", line 604, in main
E                       process()
E                     File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/worker.py", line 596, in process
E                       serializer.dump_stream(out_iter, outfile)
E                     File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/serializers.py", line 259, in dump_stream
E                       vs = list(itertools.islice(iterator, batch))
E                     File "/opt/conda/envs/py37/lib/python3.7/site-packages/bigdl/orca/data/utils.py", line 496, in farrow
E                       pd_df = table.to_pandas()
E                     File "pyarrow/array.pxi", line 757, in pyarrow.lib._PandasConvertible.to_pandas
E                     File "pyarrow/table.pxi", line 1740, in pyarrow.lib.Table._to_pandas
E                     File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyarrow/pandas_compat.py", line 768, in table_to_blockmanager
E                       from pandas.core.internals import BlockManager
E                   ModuleNotFoundError: No module named 'pandas'
~~~
test link:
https://github.com/intel-analytics/BigDL/actions/runs/3879446514/jobs/6616638679

Reply:
tfpark needs h5py, make sense.

Reply:
`test_estimator_keras_for_spark.py` Tests in this file needs pandas, make sense.